File: modules/led/led-onoff/blink.py
import RPi.GPIO as GPIO
import time
def blink(pin):
	GPIO.output(pin,GPIO.HIGH)
	time.sleep(1)
	GPIO.output(pin,GPIO.LOW)
	time.sleep(1)
	return

channel = 18
GPIO.setmode(GPIO.BCM)
GPIO.setup(channel,GPIO.OUT)
for i in range(10):
	blink(channel)
GPIO.cleanup()

File: modules/led/led-onoff/blink.c
#include <stdio.h>
#include <stdlib.h>
#include <wiringPi.h>
int main (int argc, char **argv) {
	printf ("Raspberry Pi blink\n") ;
	if (wiringPiSetup () == -1)
		return 1 ;
	int gpio = atoi(argv[1]);
	printf("gpio : %d\n",gpio);
	pinMode (gpio, OUTPUT) ;         // aka BCM_GPIO pin 17
	int i;
	for (i=0;i<10;i++) {
		digitalWrite (gpio, HIGH) ; delay (500) ;           
		digitalWrite (gpio, LOW) ; delay (500) ;
	}
	return 0 ;
}
